FF1Lib/FFRVersion.cs

```diff
@@ -3,9 +3,9 @@
 	// THIS FILE IS UPDATED DURING THE BUILD AND DEPLOY. DO NOT EDIT UNLESS YOU KONW WHAT YOU ARE DOING.
 	public static class FFRVersion
 	{
-		public const string Version = "3.1.1";
-		public const string Sha = "SHA";
-		public const string Branch = "BRANCH";
-		public const string MasterBranch = "master";
+		public static readonly string Version = "3.1.1";
+		public static readonly string Sha = "SHA";
+		public static readonly string Branch = "BRANCH";
+		public static readonly string MasterBranch = "master";
 	}

```

Comment:
What were the warnings here?

Reply:
The warnings were actually about unreachable code doing an if check on what the value was because it was constant and thus "always false" 

Of course we have a script that edits these....

Reply:
Hah, cool.